### PR TITLE
Added Automatic module name to the manifest

### PR DESCRIPTION
**Pull request by sukanth**
fix for issue https://github.com/spring-projects/spring-batch/issues/867

decompiled the jar to makesure that the manifest has Automatic-Module_Name 
image

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -229,6 +230,7 @@
 				<artifactId>maven-javadoc-plugin</artifactId>
 				<version>${maven-javadoc-plugin.version}</version>
 				<configuration>
+					<sourcepath>src/main/java</sourcepath>
```
Comment (fmbenhassine) [inline pom.xml:233]: Is this related to the addition of the automatic module name?

**Comment by fmbenhassine** [comment]
Thank you for your PR! When I compare the newly added module name `org.springframework.batch` to other spring projects, I find it inconsistent. Here are some examples:

| Module  | Automatic-Module-Name |
| :------------- | :------------- |
| spring-core  | spring.core  |
| spring-jms  | spring.jms  |
| spring-data-jpa  | spring.data.jpa  |
| spring-integration-core  | spring.integration.core  |

So for `spring-batch-core`, it should be `spring.batch.core` and not `org.springframework.batch`. Same thing for other modules. Could you please update the PR to make the naming consistent with other projects? Thank you upfront.

--

NB: please rename the branch from `main` to `GH-867` for your PR.

File: pom.xml (hunk 1 above)
Comment (sukanth) [reply pom.xml:233]: yes it is

**Comment by sukanth** [comment]
> Thank you for your PR! When I compare the newly added module name `org.springframework.batch` to other spring projects, I find it inconsistent. Here are some examples:
> 
> Module	Automatic-Module-Name
> spring-core	spring.core
> spring-jms	spring.jms
> spring-data-jpa	spring.data.jpa
> spring-integration-core	spring.integration.core
> So for `spring-batch-core`, it should be `spring.batch.core` and not `org.springframework.batch`. Same thing for other modules. Could you please update the PR to make the naming consistent with other projects? Thank you upfront.
> 
> --
> 
> NB: please rename the branch from `main` to `GH-867` for your PR.

Sure , i will work on it and update the PR and the Branch name..

File: pom.xml (hunk 1 above)
Comment (sukanth) [reply pom.xml:234]: this is what happens when i remove sourcepath and build the project
image

**Comment by sukanth** [comment]
Renamed Branch to GH-867 https://github.com/spring-projects/spring-batch/pull/4444

**Comment by sukanth** [comment]
> Automatic-Module-Name

Changed the module name appropriately in this PR https://github.com/spring-projects/spring-batch/pull/4444/

**Pull request closed without merging**